editor/src/clj/editor/pipeline/bob.clj

```diff
@@ -53,7 +53,7 @@
   (catch Exception e
     (throw (ex-info "Failed to set verbose logging field in Bob." {} e))))
 
-(def skip-dirs #{".git" "build/default" ".internal"})
+(def skip-dirs #{".git" "build" ".internal" })
```

Comment:
since it's impossible to change build folder from the editor we can replace "build/default" with "build"

Reply:
LGTM! I can probably copy this change to `editor-dev` branch

Reply:
sure, but it will be just a half of the fix, because it is broken when build from bob (without the editor) as well

Reply:
Yes, you are right